### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by samikuikka**
Start of the routing in the project.

### CODE REVIEW

**Review by itulkki — APPROVED** [review]
Looks good. This will require modifications to the back-end since going directly to /about page won't be handled correctly by express.static().

File: frontend/src/components/PromptCategoryBox.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const PromptCategoryBox: FC<PromptCategoryBoxProps> = ({
     setCategory,
 }) => {
     return (
-        <div className="w-full flex flex-col min-h-fit justify-start items-center mt-4 mb-4">
```
Comment (KorpelainenTommi) [inline frontend/src/components/PromptCategoryBox.tsx:16]: Why was this changed?

**Review by KorpelainenTommi — APPROVED** [review]
Good work. The pull request has a minor change to component margins that seems irrelevant, but personally I'm ready to approve it.

File: frontend/src/components/PromptCategoryBox.tsx (hunk 1 above)
Comment (samikuikka) [reply frontend/src/components/PromptCategoryBox.tsx:16]: While changing the top-app-bar to sticky I found a bug (content and top-app-bar were on top of each other even without scrolling) that was caused by margins in the code, so I started deleting unnecessary ones.
This was probably one that got accidentally deleted by this "bug-hunting" 🐛.
If needed we can add that line back to original if it looks better that way.

**Review by jaakkoha-bytecraft — APPROVED** [review]
